client/src/components/Calendar/index.js

```diff
@@ -41,17 +41,17 @@ class BigCalendar extends Component {
       });
   }
 
-  click = (event) => {
+  click = event => {
```

Comment:
I don't think you have to do it here, but just a heads up that maybe these functions could have better names?

`click` => `bookNewEvent`
`show` => `seeEventDetails`

Also, out of interest, what does the second argument `event` do in `history.push`?

Reply:
I guess it passes props to the pathname component or something? 

Also, I guess it comes from React Big Calendar, sah? When you click on something it passes the arguments - should it definitely be called `event` in this case, and not something like `dates`? It makes sense if it's in the `onSelectEvent` for it to be `event` but this function is assigned to `onSelectSlot`

Reply:
Sorry to clarify this second comment, I'm asking what is the argument that this `click` function receives, which you call `event` here?

Is it the normal information about the clickEvent itself like [this](https://developer.mozilla.org/en-US/docs/Web/Events/click)? Or is it special info that ReactBigCalender gives to these actions

Reply:
If we `console.log(event)` what does it look like

Reply:
the event details we got from the GET /events endpoint
thats what we got by console.log("event",event):
event {id: 10, title: "kkkkk", start: Mon Feb 18 2019 09:49:20 GMT+0200 (Israel Standard Time), end: Tue Feb 19 2019 09:49:20 GMT+0200 (Israel Standard Time), status: 1, …}

Reply:
"`click` => `bookNewEvent`
`show` => `seeEventDetails` "
changed

Reply:
> the event details we got from the GET /events endpoint
> thats what we got by console.log("event",event):
> `event {id: 10, title: "kkkkk", start: Mon Feb 18 2019 09:49:20 GMT+0200 (Israel Standard Time), end: Tue Feb 19 2019 09:49:20 GMT+0200 (Israel Standard Time), status: 1, …}`

Is this when you console log it inside the `bookNewEvent` function or `seeEventDetails` - would it not be different for each?

Reply:
Sorry, I don't want to hold this PR up - if you want to merge it, go ahead, we can talk about this tomorrow